engine/class_modules/sc_hunter.cpp

```diff
@@ -1249,6 +1253,9 @@ struct hunter_action_t: public Base
     affected_by.lead_from_the_front = parse_damage_affecting_aura( this, p->talents.lead_from_the_front_buff );
 
     affected_by.tww_s2_mm_2pc = parse_damage_affecting_aura( this, p -> tier_set.tww_s2_mm_2pc->effectN( 2 ).trigger() );
+    affected_by.tww_s2_sv_2pc = parse_damage_affecting_aura( this, p -> tier_set.tww_s2_sv_2pc ); // Winning Streak affects Wildfire Bomb
```

Comment:
these will need to look at the buff spelldata instead, as those have the effects that apply the damage bonus mods to the list of spells